src/definitions/modules/progress.less

```diff
@@ -87,6 +87,17 @@
   text-align: @progressTextAlign;
 }
 
+& when (@variationProgressRightAligned) {
+  .ui.right.aligned.progress {
+    display: flex;
+    justify-content: flex-end;
+    & .bar > .progress {
+      left: @progressRightAlignedLeft;
+      right: @progressRightAlignedRight;
+    }
+  }
```

Comment:
I think it's better to specify `display: flex` to `ui.progress` by default instead of changing for `right.aligned`. Basically, it should be the same display type and the direction can be styled with `justify-content: flex-end` as right aligned and `justify-content: flex-start` which will be default and can be ignored.

Reply:
Very good idea 👍  I adjusted it accordingly by 198168aa0eaf4857d53b4554038242f18591442b
Please review again